src/platform/forms-system/src/js/review/ReviewCollapsibleChapter.jsx

```diff
@@ -415,8 +415,10 @@ class ReviewCollapsibleChapter extends React.Component {
   render() {
     let pageContent = null;
 
-    const chapterTitle = this.getChapterTitle(this.props.chapterFormConfig);
-
+    let chapterTitle = this.getChapterTitle(this.props.chapterFormConfig);
+    if (chapterTitle === 'Disabilities') {
```

Comment:
is there a way to do this without string matching manually swaping out?  maybe updating the source of turth?

Reply:
I'll take a look at that suggesting.  But this can move into the getChapterTitle function as it's logic should go into it

Reply:
Fixed upstream